MovieSharing/APIManager.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  VideoRequest.swift
+//  MovieSharing
+//
+//  Created by Mariam Korashi on 29.09.21.
+//
+
+import Foundation
+
+class APIManager {
+    
+    // MARK: - Types
+    
+    enum VideoError: Error {
+        case noDataAvailable
+        case canNotProcessData
+        case emptyData
+    }
+
+    // MARK: - Public API
+    
+    func getVideos (completion: @escaping (Result<[Video],VideoError>) -> Void) {
```

Comment:
- [ ] remove single space after method name before parameter

- [ ] we don't use get/set keywords in our method names (only in computed properties) as in certain cases this could cause compile issues. since we are accessing data from a server here, our convention is to use `fetch`, so consider renaming to `fetchVideos`